core/commands/repository/interactors/regenerate_repository_upload_token.py

```diff
@@ -0,0 +1,25 @@
+import uuid
+
+from codecov.commands.base import BaseInteractor
+from codecov.commands.exceptions import ValidationError
+from codecov.db import sync_to_async
+from codecov_auth.models import Owner
+from core.models import Repository
+
+
+class RegenerateRepositoryUploadTokenInteractor(BaseInteractor):
+    @sync_to_async
+    def execute(self, repo_name: str, owner_username: str) -> uuid:
```

Comment:
is uuid a type? it looks like we're setting it to the library itself

Reply:
updated